php/php.api.phpmodule/src/org/netbeans/modules/php/api/PhpVersion.java

```diff
@@ -136,6 +136,39 @@ public boolean hasNamespaces() {
         return namespaces;
     }
 
+    /**
+     * Check whether this version supports scalar and return type declarations.
+     *
+     * @return {@code true} if this version scalar and return type declarations,
+     * {@code false} otherwise
+     * @since 2.67
+     */
+    public boolean hasScalarAndReturnTypes() {
+        return this.compareTo(PhpVersion.PHP_70) >= 0;
```

Comment:
Nice solution! I was thinking about defining this method in each `PhpVersion` item or using a boolean flag but your solution is much better!